### PR TITLE
OP-8: recurrent image checks

### PR DESCRIPTION
**Pull request by jkapponms**

### CODE REVIEW

File: charts/opennms-operator/values.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ operator:
   serviceName: opennms-operator
   image: opennms-operator:bleeding
   imagePullPolicy: IfNotPresent
+  imageUpdateFreq: 60
```
Comment (jbeuree) [inline charts/opennms-operator/values.yaml:6]: The code appears to use this as minutes. Can at least a comment be added here so this is more obvious

File: config/.env [hunk 2]
```diff
@@ -1,2 +1,3 @@
-VERSION=0.0.1
-DEFAULT_OPENNMS_VALUES_FILE=./charts/opennms/values.yaml
\ No newline at end of file
+VERSION=0.0.2
+DEFAULT_OPENNMS_VALUES_FILE=./charts/opennms/values.yaml
+IMAGE_UPDATE_FREQUENCY=60
```
Comment (jbeuree) [inline config/.env:3]: Maybe another comment here as this looks to set the same timer

File: internal/image/image_check.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	"github.com/go-logr/logr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"time"
+)
+
+const (
+	None             = "none"
+	Now              = "now"
+	Autoupdate       = "autoupdate"
```
Comment (jbeuree) [inline internal/image/image_check.go:15]: Suspect this is no longer needed

Comment (jkapponms) [reply internal/image/image_check.go:15]: I figured. I'd removed it from other places but forgot to here.

File: config/.env (hunk 2 above)
Comment (jkapponms) [reply config/.env:3]: Only the above should need a comment, as it's what's setting this value in the actual runtine ENV

**Review by jbeuree — DISMISSED** [review]
As it looks to be a work in progress, those minor comments can be included in the next set of changes

**Pull request merged**